### PR TITLE
Update support page

### PR DESCRIPTION
**Pull request by malinatran**
Changes include fundraising goal and conforming to style guide.

One thing I'd like to suggest: @JoaquinTrinanes @davidvanbeveren thoughts on making the progress chart be in USD since that's what we're using everywhere?

### CODE REVIEW

File: src/pages/support.tsx [hunk 1]
```diff
@@ -138,17 +143,17 @@ const Support: React.FC<{ patrons: string[]; patreonFunding: number }> = ({
...
         <span className="font-bold">monthly donation via Patreon</span> as that
-        gives us the most stability every month, but we also accept one time
-        donations via Paypal. If you would like to make a larger contribution of
-        $1000 or more (thank you!) please contact us for other options like wire
-        or check. We have a fiscal sponsor that will let you claim your donation
-        as tax deductable!
+        gives us the most stability every month, but we also accept one-time
+        donations via PayPal. If you would like to make a larger contribution of
+        $1,000 or more (thank you!), please{' '}
+        <Link href="mailto:hello@veganhacktivists.org">contact us</Link> for
```
Comment (malinatran) [inline src/pages/support.tsx]: Is there a way to make this link stand out? 

It currently blends in with the rest of the text so it's not clear that people can click on it. Nbd if not, just a nice-to-have.

Comment (JoaquinTrinanes) [reply src/pages/support.tsx]: Sure! We have a `<CustomLink>` component that does that (with a predefined style), and if you want something different you can add an `<a>` element:

```
<Link>
    <a add styles here!>
        Text
    </a>
</Link>
```

Comment (malinatran) [reply src/pages/support.tsx]: Thank you, updated!

**Comment by JoaquinTrinanes** [comment]
> One thing I'd like to suggest: @JoaquinTrinanes @davidvanbeveren thought son making the progress chart be in USD since that's what we're using everywhere?

Technicaly it's totally feasible, we're currently getting the funding amount from the CMS instead of the API so it's as easy as changing the number and euro symbols. We have means to retrieve the data live from patreon but I'd need to cross check with David the amount is correct.

We can also use the CMS value as 'other donations' and sum it to the patreon one.

**Review by davidvanbeveren — APPROVED** [review]

**Pull request merged**